apps/verticalhome/js/pin_apps.js

```diff
@@ -62,12 +87,12 @@
         manifest: null,
         index: this.index
       };
-      this.element.removeEventListener("click", this);
+      this.element.removeEventListener('click', this);
       this.targetApp = null;
     },
 
     launch: function() {
-      if(!this.targetApp) return;
+      if(!this.targetApp) {return;}
```

Comment:
return on a new line? :)
Always thought that constructions like
if () {
something_here;
}

look better than if () { something_here; }